app/jobs/send_email_job.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SendEmailJob < ActionMailer::MailDeliveryJob
+  queue_as 'webhook'
```

Comment:
why `webhook` here?

Reply:
Well, I wanted to move out of the `default` one. But maybe we should add an `email` one